docker/Dockerfile

```diff
@@ -25,15 +24,16 @@ ENV USERS_DB=/config/.users.yml
 ENV UVICORN_PORT=6900
 
 RUN useradd -ms /bin/bash worker
-COPY scripts/start_argilla_server.sh /home/worker
-COPY --chown=worker:worker --from=builder /my-virtual-env /home/worker/my-virtual-env
-ENV PATH="/home/worker/my-virtual-env/bin:$PATH"
 
 # Create argilla volume
 RUN mkdir -p "$ARGILLA_HOME_PATH" && \
-    chown worker:worker "$ARGILLA_HOME_PATH"
+  chown worker:worker "$ARGILLA_HOME_PATH"
 VOLUME $ARGILLA_HOME_PATH
 
+COPY scripts/start_argilla_server.sh /home/worker
+COPY --chown=worker:worker --from=builder /opt/venv /opt/venv
```

Comment:
```suggestion
# Destination folder must be the same as the builder one. Otherwise installed script won't work (since the installation fixes the path inside the script
COPY --chown=worker:worker --from=builder /opt/venv /opt/venv
```